src/compiler/commandLineParser.ts

```diff
@@ -56,6 +56,14 @@ namespace ts {
             category: Diagnostics.Command_line_Options,
             description: Diagnostics.Stylize_errors_and_messages_using_color_and_context_experimental
         },
+        {

```

Comment:
I would rather we do not add a new flag if we can. i think `auto` is just `pretty === undefined`. 

if we really need a new value i would make pretty take `boolean | "auto"`

Reply:
`boolean | "auto" | "simple" | "styled"`? (Where boolean toggles between `auto` and `styled`).